deploy/crds/bundle.yaml

```diff
@@ -3941,9 +4035,16 @@ spec:
                           type: object
                         engineVersion:
                           default: v2
+                          description: EngineVersion specifies the template engine version that should be used to compile/execute the template specified in .data and .templateFrom[].
```

Comment:
It should come from the deletion of the blank line
[Ref](https://github.com/external-secrets/external-secrets/pull/2819/files#diff-24c732d9db01bb7a0d8b82c495448fa8c0bba991add755685d943ff6ff67ae33L94)